src/Parsing/Impl/Parser.cs

```diff
@@ -335,7 +335,12 @@ public class Parser {
                 return n;
             }
 
-            ReportSyntaxError("syntax error");
+            if (_lookahead.Token.Kind == TokenKind.NewLine) {
+                // Incomplete member expression, report dot, don't point to new line.
+                ReportSyntaxError(prevTokenStart, prevTokenStart + prevTokeLength, "syntax error");
```

Comment:
Previously, the behavior without a comment is to squiggle at the character right after the dot; is it possible to do that again without the squiggle showing after a comment? I think with this, the squiggle shows right on top of the dot as though it's the issue rather than missing something following.